lispy.el

```diff
@@ -8563,14 +8563,19 @@ When ARG is non-nil, unquote the current string."
 
 (defvar lispy-mode-map-evilcp
   (let ((map (copy-keymap lispy-mode-map-base)))
-    (define-key map (kbd "M-)") 'lispy-close-round-and-newline)
+    ;; pairs
     (define-key map (kbd "[") 'lispy-open-square)
     (define-key map (kbd "]") 'lispy-close-square)
     (define-key map (kbd "{") 'lispy-open-curly)
     (define-key map (kbd "}") 'lispy-close-curly)
     (define-key map (kbd ")") 'lispy-right-nostring)
     (define-key map (kbd "\"") 'lispy-doublequote)
+    ;; wrapping
```

Comment:
wrapping commands are backed by lispy, not by evil-cleverparens or paredit (no need to install these)